backend/dep/wire.go

```diff
@@ -137,6 +140,38 @@ func InjectEnv() env.Env {
 	return env.GoDotEnv{}
 }
 
+// InjectGRPCService creates gRPC service with configured dependencies.
+func InjectGRPCService(
+	runtime env.Runtime,
+	prefix provider.LogPrefix,
+	logLevel logger.LogLevel,
+	sqlDB *sql.DB,
+	securityPolicy security.Policy,
+	dataDogAPIKey provider.DataDogAPIKey,
+) (service.GRPC, error) {
+	wire.Build(
+		wire.Bind(new(timer.Timer), new(timer.System)),
+		wire.Bind(new(repository.ShortLink), new(sqldb.ShortLinkSQL)),
+		wire.Bind(new(rpc.API), new(grpcapi.ShortGRPCApi)),
+		wire.Bind(new(shortlink.MetaTag), new(shortlink.MetaTagPersist)),
+
+		observabilitySet,
+
+		sqldb.NewShortLinkSQL,
+		shortlink.NewMetaTagPersist,
+
+		grpcapi.NewMetaTagServer,
+		grpcapi.NewShortGRPCApi,
```

Comment:
```suggestion
		grpcapi.NewShort,
```
The package name already mentioned grpcAPI
I think this is adapter and can be grouped with `shortlink.NewMetaTagPersist`